doc/locale/ru/LC_MESSAGES/cartridge_kubernetes_guide/index.po

```diff
@@ -0,0 +1,807 @@
+
+msgid "Tarantool Cartridge on Kubernetes"
```

Comment:
I'm confused. Shouldn't the translation result be in the .po files?

Reply:
Translation can be in .po files and only there. But translation is done by humans, and they couldn't do it yet, and won't be until this PR gets merged.